qemu/tests/x86_cpu_test_0x40000001.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from virttest import error_context
+from virttest import data_dir
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check kvm 0x40000001 inside guest.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+    nums_cpu = str(vm.cpuinfo.smp)
+
+    src_cpuid = os.path.join(data_dir.get_deps_dir(), "cpuid.src.tar.gz")
+    vm.copy_files_to(src_cpuid, '/home/')
+    try:
+        session.cmd('tar -xzf /home/cpuid.src.tar.gz -C /home/ && cd /home/cpuid-20201006 && make')
+        results = session.cmd_output('cd /home/cpuid-20201006 && ./cpuid -r | grep eax=0x40000001 | uniq -c').strip()
```

Comment:
Also recommend a variable for the command `cmd = './cpuid -r | grep eax=0x40000001 | uniq -c'`

Reply:
> Also recommend a variable for the command `cmd = './cpuid -r | grep eax=0x40000001 | uniq -c'`

Thanks and update.